docs/governance/index.md

```diff
@@ -6,6 +6,8 @@ sidebar_position: 9
 
 This page needs contributions. Help us write it!
 
+Governance in web3 is driven by the community. A good governance will iron-out all those hiccups that may arise due to a change in market, contract or in technology.
```

Comment:
`iron-out all those hiccups that may arise due to a change in market, contract or in technology`
Let's try to keep it as objective as possible.

Also let's include ways on how people can get involved in Stacks governance.